### PR TITLE
#165853787 Notify attendees on slack when host updates an event

### PR DESCRIPTION
**Pull request by Alameen688**
#### What Does This PR Do?

- This PR implements slack notifications for all attendees for an event when the host changes the venue or date for the event
#### Description Of Task To Be Completed
- [x] Notify attendees on slack when the venue, date/time for an event changes
- [x] Notify attendees on slack when an event is canceled
#### Any Background Context You Want To Provide?

- N/A
#### How can this be manually tested?

- Create an event
- Add yourself to the attendee list, you could do that by manually adding the info to the api_attend table
- Update an event
- You should receive a slack notification

#### What are the relevant pivotal tracker stories?

[#165853787](https://www.pivotaltracker.com/story/show/165853787)

#### Screenshot(s)
Screen Shot 2019-05-14 at 5 07 35 AM

### CODE REVIEW

File: server/graphql_schemas/utils/helpers.py [hunk 1]
```diff
@@ -178,3 +180,16 @@ def upload_image_file(uploaded_file):
         url = "http://localhost:8000/static{}".format(
             fs.url(filename))
         return url
+
+
+async def send_bulk_update_message(event_instance, message, notification_text):
```
Comment (olusoladavid) [inline server/graphql_schemas/utils/helpers.py:186]: Great job on this implementation @Alameen688. Is it possible to debounce the sending of each message by 1 second at least? The `chat.postMessage` method is rate-limited by 1 message per second on Slack. Please see: https://api.slack.com/docs/rate-limits#tier_t5

**Review by olusoladavid — COMMENTED** [review]
Great job on this implementation @Alameen688. Works as required. I made a key comment. Please check it out. Cheers 🥂

File: server/graphql_schemas/utils/helpers.py (hunk 1 above)
Comment (Alameen688) [reply server/graphql_schemas/utils/helpers.py:186]: Thanks for pointing that out. I'm on it

**Review by olusoladavid — APPROVED** [review]
Great job on this implementation @Alameen688. Works as required.

**Pull request merged**